plugins/Delay/DelayControlsDialog.cpp

```diff
@@ -22,10 +22,13 @@
  *
  */
 
-#include "delaycontrolsdialog.h"
-#include "delaycontrols.h"
+#include "DelayControlsDialog.h"
+#include "DelayControls.h"
 #include "embed.h"
 #include "TempoSyncKnob.h"
+#include "../Eq/EqFader.h"
```

Comment:
Well, we need to rewrite the Fader class anyway in 2.0, so that we can make fx faders be in dBV scale. Probably/hopefully at that point we can consolidate all the different faders into one class... so for now maybe we don't really need to worry too much about it?